changelog.txt

```diff
@@ -5,6 +5,7 @@
  * Following methods in managed RealmList now throw IllegalStateException instead of ArrayIndexOutOfBoundsException when RealmList.isValid() returns false: set(int,RealmObject), move(int,int), remove(int), get(int)
  * Following methods in managed RealmList now throw IllegalStateException instead of returning 0/null when RealmList.isValid() returns false: clear(), removeAll(Collection), remove(RealmObject), first(), last(), size(), where()
  * RealmPrimaryKeyConstraintException is now thrown instead of RealmException if two objects with same primary key are inserted.
+ * Changed return type of RealmProxyMediator.getModelClasses() to Set<Class<? extends RealmObject>>.
```

Comment:
In the changelog we only list changes visible to users. So I think you should just state the problem that this PR fixes.